thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/pojo/MergedAnomalyResultBean.java

```diff
@@ -23,6 +23,24 @@
   private Long createdTime;
   private String message;
   private boolean notified;
+  private double avgCurrentVal;
+  private double avgBaselineVal;
```

Comment:
Add these two fields: avgCurrentVal and avgBaselineVal, to hashCode, equals, compareTo, and toString methods.